app/modules/events/utils.tsx

```diff
@@ -0,0 +1,13 @@
+export function groupDataPatcher(data: any) {
```

Comment:
Since there is no JSX in here, maybe just a `.ts` file instead of `tsx` (which kind of suggests there is React/JSX code in the file)